client/me/account/main.jsx

```diff
@@ -890,8 +884,9 @@ const Account = createReactClass( {
 				<ReauthRequired twoStepAuthorization={ twoStepAuthorization } />
 				<FormattedHeader brandFont headerText={ translate( 'Account Settings' ) } align="left" />
 
+				<SectionHeader label={ translate( 'Account Information' ) } />
```

Comment:
Nitpick: I have the feeling that we lately prefer a sentence-case format (`Account information`) over a title-case format (`Account Information`): https://github.com/Automattic/wp-calypso/pulls?q=is%3Apr+%22sentence+case%22+is%3Aclosed+

Reply:
👍 Pushed a fix.